Description: Use static libcheck version.
Author: Mihai Moldovan <ionic@ionic.de>

Within Debian, only the static libcheck library is shipped. Use it correctly,
since the static library requires additional private libraries and definitions.

--- a/tests/hawkey/CMakeLists.txt
+++ b/tests/hawkey/CMakeLists.txt
@@ -28,7 +28,8 @@ add_executable(test_hawkey_main ${hawkey
 set_target_properties(test_hawkey_main PROPERTIES COMPILE_FLAGS -fPIC)
 target_link_libraries(test_hawkey_main
     libdnf
-    ${CHECK_LIBRARIES}
+    ${CHECK_STATIC_LIBRARIES}
+    ${CHECK_STATIC_LDFLAGS}
     ${SOLV_LIBRARY}
     ${SOLVEXT_LIBRARY}
     ${RPMDB_LIBRARY}
